package.json

```diff
@@ -30,48 +30,14 @@
     "dist/*.js"
   ],
   "devDependencies": {
-    "@babel/core": "^7.6.2",
-    "@babel/preset-env": "^7.6.2",
-    "babel-preset-es2015-rollup": "^3.0.0",
-    "clean-css": "^4.2.1",
-    "coveralls": "^3.0.0",
-    "eslint": "^6.0.0",
-    "eslint-config-chartjs": "^0.2.0",
-    "eslint-plugin-html": "^6.0.0",
-    "gitbook-cli": "^2.3.2",
-    "gulp": "^4.0.0",
-    "gulp-eslint": "^6.0.0",
-    "gulp-file": "^0.4.0",
-    "gulp-htmllint": "^0.0.16",
-    "gulp-jsdoc3": "^2.0.0",
-    "gulp-replace": "^1.0.0",
-    "gulp-size": "^3.0.0",
-    "gulp-streamify": "^1.0.2",
-    "gulp-terser": "^1.1.6",
-    "gulp-zip": "^5.0.0",
-    "jasmine": "^3.3.0",
```

Comment:
We probably need a lot of this stuff to make the tests pass. I notice that Travis is currently failing

Reply:
i started from 0 and only added what was necessary to get the build working. i have no interest in getting the rest of your setup working - i trust you guys can handle it from this point if you like what you see here.